Header header
uint32 nActiveEngineID
string strActiveEngineDescription

